src/components/Countdown/countdownHooks/useCountdown.js

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from "react";
+
+const useCountdown = (targetDate) => {
+  const countDownDate = new Date(targetDate).getTime();
+
+  const [countDown, setCountDown] = useState(
+    countDownDate - new Date().getTime()
+  );
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      setCountDown(countDownDate - new Date().getTime());
+    }, 1000);
+
+    return () => clearInterval(interval);
+  }, [countDownDate]);
+
+  return getReturnValues(countDown);
+};
+
+const getReturnValues = (countDown) => {
```

Comment:
we can use separate file for these calculations. eg: utils.js keep it main folder

Reply:
Sure. I'll do it.

Reply:
done